docs/reference/useful_links.md

```diff
@@ -25,13 +25,16 @@ parent: Reference
 
 ## Blog posts
 
-* 2016-12-01 ([1](https://opensource.googleblog.com/2016/12/announcing-oss-fuzz-continuous-fuzzing.html),
-[2](https://testing.googleblog.com/2016/12/announcing-oss-fuzz-continuous-fuzzing.html),
-[3](https://security.googleblog.com/2016/12/announcing-oss-fuzz-continuous-fuzzing.html))
-* 2017-05-08 ([1](https://opensource.googleblog.com/2017/05/oss-fuzz-five-months-later-and.html),
-[2](https://testing.googleblog.com/2017/05/oss-fuzz-five-months-later-and.html),
-[3](https://security.googleblog.com/2017/05/oss-fuzz-five-months-later-and.html))
-* 2018-11-06 ([1](https://security.googleblog.com/2018/11/a-new-chapter-for-oss-fuzz.html))
+* 2016-12-01 - Announcing OSS-Fuzz: Continuous fuzzing for open source software
+([Open Source](https://opensource.googleblog.com/2016/12/announcing-oss-fuzz-continuous-fuzzing.html),
```

Comment:
Can leave all three blogs here, just for the reference :)

Reply:
Done!